Fix Version: Famatech.AdvancedIPScanner added Scopes

CodyRWhite opened the pull request:
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Updated the application to allow scope selection. 
Also updated manifest to 1.1.0



###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38004)

CodyRWhite committed:
Updated version

Updated current installer

Updated manifest version
Updated format for mutiple releases
added machine and user scopes

CodyRWhite committed:
Merge pull request #1 from CodyRWhite/development

App Update - AdvancedIPScanner scope change

OfficialEsco commented on manifests/f/Famatech/AdvancedIPScanner/2.5.3850/Famatech.AdvancedIPScanner.installer.yaml:
```diff
@@ -2,10 +2,20 @@
 PackageVersion: 2.5.3850
+Platform:
+- Windows.Desktop
+InstallerType: exe
```
This is redundant when the the installer says its inno

CodyRWhite replied:
Removed lines 5-7.

OfficialEsco replied:
Oops didnt mean to mark Platform, did it a bit quick

CodyRWhite replied:
no worries added the platform back.

OfficialEsco commented on manifests/f/Famatech/AdvancedIPScanner/2.5.3850/Famatech.AdvancedIPScanner.installer.yaml:
```diff
@@ -2,10 +2,20 @@
+InstallerType: exe
+UpgradeBehavior: install
 Installers:
 - Architecture: neutral
+  Scope: user
+  InstallerType: inno
+  InstallerUrl: https://download.advanced-ip-scanner.com/download/files/Advanced_IP_Scanner_2.5.3850.exe
+  InstallerSha256: 87BFB05057F215659CC801750118900145F8A22FA93AC4C6E1BFD81AA98B0A55
```
You need to define a custom installer switch for this to install to the user

OfficialEsco replied:
Is this the mode you are trying to add?
![image](https://user-images.githubusercontent.com/15158490/146459326-3aac8f23-bb41-47b1-91c5-062f395e998c.png)
You need to find the Installer variable for that, which might be tricky since its for Portable version and not /CURRENTUSER

CodyRWhite replied:
No. the original installer was setup with no scope. So when attempting to install with a scope set to machine it said no matching package was found. 

So I assumed no packages were required for user as that is the default scope if none is defined. My assumption may be incorrect. 

I have added some switches. I tested /CURRENTUSER and it did not install into appdata. It still installed in Program Files.

CodyRWhite replied:
Realistically there might not be a user scope available for this app. Perhaps I remove that section and just set the default scope to machine?

OfficialEsco replied:
https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/installer.md#scope
I think as long as we don't have the Portable switch we might have to go with machine only ye

CodyRWhite replied:
still running reading up on inno and seeing if there is a combination we can do to run in portable mode unattended. If not I'll change it machine.

CodyRWhite committed:
Removed redundant properties

CodyRWhite committed:
Update Famatech.AdvancedIPScanner.installer.yaml

OfficialEsco commented on manifests/f/Famatech/AdvancedIPScanner/2.5.3850/Famatech.AdvancedIPScanner.installer.yaml:
```diff
@@ -9,6 +9,10 @@ Installers:
   InstallerType: inno
   InstallerUrl: https://download.advanced-ip-scanner.com/download/files/Advanced_IP_Scanner_2.5.3850.exe
   InstallerSha256: 87BFB05057F215659CC801750118900145F8A22FA93AC4C6E1BFD81AA98B0A55
+  InstallerSwitches: 
+    Silent: /VERYSILENT
+    SilentWithProgress: /SILENT
+    Log: /LOG="<LOGPATH>"
```
These are also redundant when the InstallerType is set to Nullsoft, Inno, Burn, Msi, Msix, Appx, or Wix :)
I believe this https://github.com/microsoft/winget-cli/blob/b79e4087821c993f83f7c4731bbde7aa25612c73/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L449-L481 is what defines that

CodyRWhite committed:
Added Platform back.

CodyRWhite committed:
Removed user scope

I was unable to process an install using commandline to trigger a "portable" install. While some of the commands are available in inno they were not working/enabled in this package.
